src/JWT/Builder/JwtBuilder.cs

```diff
@@ -221,17 +234,20 @@ public T Decode<T>(string token)
             if (_decoder == null)
                 TryCreateDecoder();
 
-            return _verify ? _decoder.DecodeToObject<T>(token, _secret, _verify) : _decoder.DecodeToObject<T>(token);
+            return _verify ? _decoder.DecodeToObject<T>(token, _secrets[0], _verify) : _decoder.DecodeToObject<T>(token);
         }
 
         private void TryCreateEncoder()
         {
             if (_algorithm == null)
-                throw new InvalidOperationException($"Can't instantiate {nameof(JwtEncoder)}. Call {nameof(WithAlgorithm)}.");
+                throw new InvalidOperationException(
```

Comment:
Do you think these blocks of changes are needed (here and below, all exceptions messages)?

Reply:
I'm sorry, that was rider reformatting, same as couple of modifications (related to added or removed spaces)
I reverted this part and other functions as well (`TryCreateDecoder()` & `TryCreateValidator()`)